### ISSUE TITLE
[Feature Request]Ownership Permissions Check- Disable on per folder basis

### ISSUE THREAD

**Issue by WorthlessJ** [GameServerManagers/LinuxGSM#2659]
I would like to be able to disable ownership checks on a per-folder basis. This would allow users to link multiple servers to one common folder and save on storage space. 

In my specific case I have 6KF2+4sven different servers running the same maps, and each server has it's own folder of custom maps. This change would allow me to save ~200gb of storage space.

I understand that this can be worked around by having every server under one user, and/or instancing servers, however I would prefer to keep each server with it's own user.

It would be nice if this feature functioned like stats="on" and could simply be added to the lgsm config file. 

Example variable:

disable_p_check="/home/kf2maps/"

**Comment by dgibbs64** [comment]
@WorthlessJ have you tried symbolic links? It may be possible to use a symbolic link on these custom map dirs. I am unsure if the permissions check would pick up symbolically linked dir or not. Would you be able to try out a symbolic link and see what the outcome is?

Also is there a specific reason you would prefer not the use a single user?

The change you are requesting would require some careful thought to prevent a way of new admins being lazy and bypassing the permissions which could lead to odd issues.

**Comment by WorthlessJ** [comment]
@dgibbs64 Thanks for the quick reply on this request!

Short: I have resolved the problem, as per your suggestion with symbolic links by changing my configuration around. I can also confirm symbolic links work, whereas mounts do not.

Original problem: In order to bypass user chroot jailing, I was using mounting instead of symbolic links. While the files were readable they were not owned by the user, and LGSM would recognize this.

Solution: I have changed my jails from /home/user/ to /home/ and disallowed viewing other folders with user:group permissions. My map folders reside in /home/, and have been symbolically linked to. I have not lost any functionality, my jail setup has been simplified, and I have saved storage space. I call this a win.

While I still think this could still be a helpful feature, I understand the reasoning behind not implementing it and find this an acceptable workaround. I do think the permissions aspect is one of the best features of LGSM and probably curbs countless support requests.

Why I have a user per server: Ease of use, security and futureproofing. Each server can have an individual admin with ssh/sftp access, can have different mods, can manage their own files, etc without touching other servers. Symlinking or mounting goes against this, but the storage savings are large, and the way Sven and KF2 handle custom content, users can still add other custom content in their own directories.

I will close this request now, Cheers.
